calico_kubernetes/calico_kubernetes.py

```diff
@@ -146,35 +143,29 @@ def _configure_profile(self, endpoint):
 
         Currently assumes one pod with each name.
         """
-        pod = self._get_pod_config()
-
         logger.info('Configuring Pod Profile: %s', self.profile_name)
 
         if self._datastore_client.profile_exists(self.profile_name):
-            logger.error("Profile with name %s already exists, exiting.",
-                         self.profile_name)
-            sys.exit(1)
+            logger.warning("Profile with name %s already exists.",
```

Comment:
I'd add a debug level log in the profile_exists() branch that says something like "Profile %s already exists, no work to do" or something like that.

Add an info level log in the branch where the profile doesn't exist that says we're creating a new profile named X.